TestLib/Exception/GREYFailureFormatter.h

```diff
@@ -76,7 +50,7 @@ NS_ASSUME_NONNULL_BEGIN
  * @param stackTrace     The stack trace of the executable when the error happens.
  * @param appScreenshots The dictionary of app screenshot names and image paths
  *                       when the error happens.
- * @param format         Extra message to be included in the message.
+ * @param errorDescription         Extra message to be included in the message.
```

Comment:
Fix formatting.

Reply:
It is not an issue in the actual code, just a github formatting issue. Checkout the branch and let me know if you still want it to be changed.

Reply:
```
 * @param testCase       The test case where the failure happens.
 * @param failureLabel   The label to be associated with the error.
 * @param failureName    The name to be associated with the error.
 * @param functionName   The name of the function where the error is generated.
 * @param filePath       The path to the source code file where the error happens.
 * @param lineNumber     The line number of the source code file when the error happens.
 * @param stackTrace     The stack trace of the executable when the error happens.
 * @param appScreenshots The dictionary of app screenshot names and image paths
 *                       when the error happens.
 * @param errorDescription         Extra message to be included in the message.
```

The formatting seems wonky in the raw file as well.

Reply:
Fixed